proarc-common/src/main/java/cz/cas/lib/proarc/common/export/mets/structure/MetsElement.java

```diff
@@ -281,6 +281,20 @@ public MetsElement(DigitalObject digitalObject, Object parent, MetsContext metsC
         model = MetsUtils.getModel(relsExt);
         this.elementType = Const.typeMap.get(model);
 
+
+        if (parent instanceof MetsElement) {
+            this.parent = (MetsElement) parent;
+        }
+
+        if (parent == null) {
+            this.parent = initParent();
+        }
```

Comment:
Drobnost: nedalo by se to sjednotit do if-else?

Reply:
Jo, asi dalo :-) Dříve byl tento kód na konci metody, já to jen přesouval na začátek, abych měl rodiče...